### PR TITLE
2025 release – typography – everything else [OR-1050]

### PR DESCRIPTION
**Pull request by notlee**

### CODE REVIEW

File: components/o-editorial-typography/main.scss [hunk 1]
```diff
@@ -1,6 +1,5 @@
-@import '@financial-times/o-typography/main';
+@import '@financial-times/o-private-foundation/main';
 @import '@financial-times/o-spacing/main';
```
Comment (frshwtr) [inline components/o-editorial-typography/main.scss:2]: Looks like this was missed when working on spacing.

Comment (notlee) [reply components/o-editorial-typography/main.scss:2]: 👍 Just removed that as part of this PR

Comment (notlee) [reply components/o-editorial-typography/main.scss:2]: Oh no I didn't, I different call to oSpacingByName 🙈

Comment (notlee) [reply components/o-editorial-typography/main.scss:2]: Done!

File: components/o-forms/src/scss/modifiers/_state.scss [hunk 2]
```diff
@@ -6,11 +6,8 @@
 	.o-forms-input__state {
 		// the loading spinner and tick icons for state are mismatched in terms of size/padding,
 		// the custom line-height aligns them as closely as possible (with minimal style changes)
-		@include oTypographySans(
-			$scale: -1,
-			$line-height: 1.75,
-			$include-font-family: false
```
Comment (frshwtr) [inline components/o-forms/src/scss/modifiers/_state.scss:12]: This mixin had `include-font-family` set to `false` which does not include `font-family` in this case.. `oPrivateTypographySans` will include `font-family` when the mixin is used.

Is this a correct update?

Comment (notlee) [reply components/o-forms/src/scss/modifiers/_state.scss:12]: Yeah that's fine, I've optimised for API simplicity. Its intention was to reduce bundle size slightly, but we'll see much bigger gains through o-buttons refactoring and ultimately moving to o3

File: components/o-private-foundation/main.scss [hunk 3]
```diff
@@ -19,5 +19,12 @@
 		font-display: swap;
 	}
 
-	// @todo include italic font
+	@font-face {
```
Comment (frshwtr) [inline components/o-private-foundation/main.scss:22]: comment mentions italic font but this looks like a standard definiton for financier, does italic still need including?

Comment (notlee) [reply components/o-private-foundation/main.scss:22]: It does not! Italics are not used in o2 https://github.com/Financial-Times/n-ui-foundations/blob/main/typography/main.scss

**Review by frshwtr — APPROVED** [review]

**Pull request merged**